### PR TITLE
Feature localstorage

### PR DESCRIPTION
**Pull request by nidhimundra**
@bpowers : Please review

### CODE REVIEW

File: src/kernel/kernel.ts [hunk 1]
```diff
@@ -2499,12 +2499,12 @@ export function Boot(fsType: string, fsArgs: any[], cb: BootCallback, args: Boot
 					cb(err, undefined);
 					return;
 				}
-				let writable = new bfs.FileSystem['InMemory']();
+				let writable = new bfs.FileSystem['LocalStorage']();
```
Comment (bpowers) [inline src/kernel/kernel.ts]: will you please add a boolean flag to `BootArgs` to choose between `LocalStorage` and `InMemory`?  There are different applications that use the Browsix Kernel, including the latex example as well as the shell.  We still want the shell to use `InMemory`, and only switch to `LocalStorage` when `args.UseLocalStorage` is true.

**Comment by nidhimundra** [comment]
@bpowers : Made the requested changes

File: src/kernel/kernel.ts [hunk 2]
```diff
@@ -2499,12 +2500,20 @@ export function Boot(fsType: string, fsArgs: any[], cb: BootCallback, args: Boot
 					cb(err, undefined);
 					return;
 				}
-				let writable = new bfs.FileSystem['InMemory']();
+				if (args.useLocalStorage) {
+					let writable = new bfs.FileSystem['LocalStorage']();
```
Comment (bpowers) [inline src/kernel/kernel.ts]: let bindings are scoped to the block they are in -- they will not be visible after the if statement.  I think you need 

`let writable: bfs.FileSystem; // or whatever the type is`
before the `if`, and then just `writable = new bfs.FileSystem...` inside the `if`

**Comment by bpowers** [comment]
Looks great, thanks!

**Pull request merged**